web_src/js/features/repo-issue-list.js

```diff
@@ -185,51 +185,24 @@ async function initIssuePinSort() {
   });
 }
 
-// function initArchivedLabelFilter() {
-//   const archivedLabelEl = document.querySelector('#archived-filter-checkbox');
-//   if (!archivedLabelEl) {
-//     return;
-//   }
-//
-//   const archivedLabels = $('[data-is-archived]');
-//   const archivedElToggle = () => {
-//     for (const label of archivedLabels) {
-//       toggleElem(label, archivedLabelEl.checked);
-//     }
-//   };
-//   archivedElToggle();
-//   archivedLabelEl.addEventListener('change', () => {
-//     archivedElToggle();
-//     const url = archivedLabelEl.getAttribute('data-url');
-//     window.location = (archivedLabelEl.checked) ? url : url.replace(/&archived=true/, '');
-//   });
-// }
 function initArchivedLabelFilter() {
   const archivedLabelEl = document.querySelector('#archived-filter-checkbox');
   if (!archivedLabelEl) {
     return;
   }
 
-  const archivedLabels = document.querySelectorAll('[data-is-archived]');
-  const toggleArchivedLabels = () => {
-    const isChecked = archivedLabelEl.checked;
+  const archivedLabels = $('[data-is-archived]');
```

Comment:
Why suddenly jquery for this single call that was previously not JQuery?

Reply:
yeah, we can change this.